pkg/cache/v2/listener_test.go

```diff
@@ -180,3 +184,93 @@ func TestListenerFlush(t *testing.T) {
 		assert.Equal(t, []*core_v2.SocketAddress{}, deleteListenerAddress)
 	})
 }
+
+func BenchmarkFlush(b *testing.B) {
+	t := &testing.T{}
+	test.InitBpfMap(t)
+	t.Cleanup(test.CleanupBpfMap)
+
+	listener := &listener_v2.Listener{
+		ApiStatus: core_v2.ApiStatus_UPDATE,
+		Address: &core_v2.SocketAddress{
+			Port: uint32(80),
+			Ipv4: nets.ConvertIpToUint32("192.168.127.244"),
+		},
+		FilterChains: []*listener_v2.FilterChain{
+			{
+				Filters: []*listener_v2.Filter{
+					{
+						Name: "filter1",
+						ConfigType: &listener_v2.Filter_TcpProxy{
+							TcpProxy: &filter.TcpProxy{
+								StatPrefix: "outbound|53||kube-dns.kube-system.svc.cluster.local",
+								ClusterSpecifier: &filter.TcpProxy_Cluster{
+									Cluster: "outbound|53||kube-dns.kube-system.svc.cluster.local",
+								},
+								MaxConnectAttempts: uint32(60),
+							},
+						},
+					},
+					{
+						Name: "filter2",
+						ConfigType: &listener_v2.Filter_TcpProxy{
+							TcpProxy: &filter.TcpProxy{
+								StatPrefix: "outbound|80||kube-dns.kube-system.svc.cluster.local",
+								ClusterSpecifier: &filter.TcpProxy_Cluster{
+									Cluster: "outbound|80||kube-dns.kube-system.svc.cluster.local",
+								},
+								MaxConnectAttempts: uint32(60),
+							},
+						},
```

Comment:
This can not exist in real case, usually one filterChain can point to one route/one cluster.